specification/components/examples/requests/NHAIS/acceptance-birth-min.json

```diff
@@ -0,0 +1,37 @@
+{
+  "id": "54321",
+  "address": [
+    {
+      "line": [
+        "current house name",
+        "current number or road name",
+        "current locality",
+        "current town",
+        "current county"
+      ],
+      "postalCode": "SO15 2GB"
+    }
+  ],
+  "birthDate": "1990-01-01T09:33:00",
```

Comment:
does the time for min need be be at the boundary:
birthDate": "1990-01-01T00:01:00

Reply:
as above, max/min aren't ranges but required/optional. Maybe I should rename the files?

Reply:
> as above, max/min aren't ranges but required/optional. Maybe I should rename the files?

Understood, yeah, rename may make it clearer, thanks